### PR TITLE
Change Password

### PR DESCRIPTION
**Pull request by tlacasse**
- [x] fxml

- [x] controller

### CODE REVIEW

File: src/main/java/medicaldoctor/controllers/ProfileController.java [hunk 1]
```diff
@@ -9,52 +9,49 @@
...
-            email_tf.setText(AppSession.getActiveUser().getEmail());
-            department_tf.setText(AppSession.getActiveUser().getDepartment());
-            office_tf.setText(AppSession.getActiveUser().getOfficeNum() + "");
-            employee_type_tf.setText(AppSession.getActiveUser().getType().getName());
+        User userToDisplay = null;
+        if (AppSession.getViewingUserSelection() == null) {
+            userToDisplay = AppSession.getActiveUser();
+        } else {
+            userToDisplay = AppSession.getViewingUserSelection();
+            buttonChangePassword.setManaged(false);
```
Comment (NotSalH) [inline src/main/java/medicaldoctor/controllers/ProfileController.java:43]: This makes more sense than a flag

File: src/main/java/medicaldoctor/controllers/NavigationBarController.java [hunk 2]
```diff
@@ -198,7 +198,7 @@ public void initialize(URL location, ResourceBundle resources) {
         //buttonMustHaveThisPermission(buttonMyLabRequests, Permission.ACCESS_MY_LAB_REQUESTS);
         buttonMustHaveThisPermission(buttonRegisterNewUser, Permission.REGISTER_NEW_USER);
         //buttonMustHaveThisPermission(buttonLabRequests, Permission.ACCESS_ALL_LAB_REQUESTS);
-        buttonMustHaveThisPermission(buttonLogs, Permission.ACCESS_LOGS);
+        //buttonMustHaveThisPermission(buttonLogs, Permission.ACCESS_LOGS);
```
Comment (NotSalH) [inline src/main/java/medicaldoctor/controllers/NavigationBarController.java:201]: Will logs just be a trace of all things the user searched? Are we adding logs before we submit?

File: src/main/java/medicaldoctor/backend/LoginService.java [hunk 3]
```diff
@@ -24,6 +24,21 @@ private LoginService() {
      */
     public static LoginResult checkLogin(String username, String password)
             throws Exception {
+        return checkLogin(username, password, true);
```
Comment (NotSalH) [inline src/main/java/medicaldoctor/backend/LoginService.java:27]: what does the boolean statement do? is it for logins and logouts?

File: src/main/java/medicaldoctor/controllers/NavigationBarController.java (hunk 2 above)
Comment (tlacasse) [reply src/main/java/medicaldoctor/controllers/NavigationBarController.java:201]: imma say not enough time

**Pull request merged**